tensorflow/inference/docker/1.15.3/py2/Dockerfile.cpu

```diff
@@ -0,0 +1,101 @@
+FROM ubuntu:18.04
+
+LABEL maintainer="Amazon AI"
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    ca-certificates \
+    curl \
+    git \
+    wget \
+    build-essential \
+    vim \
+ && rm -rf /var/lib/apt/lists/*
+
+# See http://bugs.python.org/issue19846
+ENV LANG C.UTF-8
+
+# Add arguments to achieve the version, python and url
+#  PYTHON=python for 2.7
+#  PYTHON=python3 for 3.5, 3.6 is not available directly on 16.04
```

Comment:
nit: These old comments should be removed

Reply:
We decided to remove the old comments in another PR as these images can be tested and modified by other author PR